FlowerinoPlugin/src/org/flowerplatform/flowerino_plugin/FlowerPlatformPlugin.java

```diff
@@ -371,5 +457,31 @@ public String editLinkedRepository(boolean showAdditionalText) {
 		writeProperties(properties, getProjectPropertiesFile());
 		return result;
 	}
+
+	public static File getBuildFolder(Sketch sketch) throws IOException {
+		SketchData sketchData = null;
+		try {
+			Field f;
+			f = sketch.getClass().getDeclaredField("data");
+			f.setAccessible(true);

```

Comment:
de cautat in tot proiectul dupa asta, sa vedem unde folosim reflexia. Si sa inlocuim cu functia util a lui andrei